eng/Versions.props

```diff
@@ -127,7 +127,7 @@
     <MonoDevelopSdkPackageVersion>1.0.15</MonoDevelopSdkPackageVersion>
     <MoqPackageVersion>4.10.0</MoqPackageVersion>
     <!-- STOP!!! We need to reference the version of JSON that our HOSTS supprt. -->
-    <NewtonsoftJsonPackageVersion>9.0.1</NewtonsoftJsonPackageVersion>
+    <NewtonsoftJsonPackageVersion>12.0.2</NewtonsoftJsonPackageVersion>
```

Comment:
Do our hosts support this version and is this rather significant breaking change in the packages we ship mandatory? This affects _every_ package using Newtonsoft.Json.

/fyi @Pilchie

Reply:
> Do our hosts support this version and is this rather significant breaking change in the packages we ship mandatory? This affects _every_ package using Newtonsoft.Json.

Hu? This only impacts tooling pieces IIRC and the only reason why we could upgrade was because we're already being binding redirected to the new version anyhow 😄.

Reply:
> This only impacts tooling pieces IIRC &hellip;

My point was this affects anything from this repo that depends on Newtonsoft.Json.

> we're already being binding redirected to the new version anyhow

What does the redirection of which you speak.

> Am I missing something ?

Maybe
> ?  STOP!!! We need to reference the version of JSON that our HOSTS supprt.

Reply:
> My point was this affects anything from this repo that depends on Newtonsoft.Json.

Yup, that's intended.

> What does the redirection of which you speak.

Visual Studio and VS4Mac (the two hosts we operate in) both are .NET Framework applications and operate on binding redirects when there are conflicting assemblies. So today when Razor bits run in VS when they attempt to load Newtonsoft.Json it respects VS' overarching binding redirect rules; so in our case it ends up loading a shared version of Newtonsoft.Json not even provided by our extension.